lib/vanagon/platform/windows.rb

```diff
@@ -107,21 +107,22 @@ def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         cg_name = "ProductComponentGroup"
         dir_ref = "INSTALLDIR"
+        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} -ext WiXUtilExtension -ext WixUIExtension"
```

Comment:
I have gone through the code (including history) and have determined that although the following variables are defined in windows.rake, the values are not used by the installer:
```
  flags['PuppetDescTag'] = describe 'downloads/puppet'
  flags['FacterDescTag'] = describe 'downloads/facter'
  flags['HieraDescTag']  = describe 'downloads/hiera'
  flags['MCODescTag']  = describe 'downloads/mcollective'
```
```puppet.wxi``` does reference two of them (?define), but again, doesn't use them.
So I think we can safely ignore these.

/cc @rob @iristyle @jpogran